app/src/main/kotlin/io/homeassistant/companion/android/player/HAMediaPlayer.kt

```diff
@@ -0,0 +1,377 @@
+                modifier = Modifier
+                    .align(Alignment.Center)
+                    .size(64.dp),
+            )
+        }
+
+        Controls(
+            player = player,
+            isBuffering = bufferingState.isBuffering,
+            showControls = showControls,
+            isFullScreen = isFullscreen,
+            onClickFullscreen = {
+                isFullscreen = !isFullscreen
+            },
+        )
+    }
+}
+
+@Composable
+private fun BoxScope.Controls(
+    player: Player,
+    isBuffering: Boolean,
+    showControls: Boolean,
+    isFullScreen: Boolean,
+    onClickFullscreen: () -> Unit,
+) {
+    val animatedAlpha by animateFloatAsState(
+        targetValue = if (showControls) 1.0f else 0f,
+        label = "ControlsAlpha",
+    )
+    val modifier = Modifier.graphicsLayer {
+        alpha = animatedAlpha
+    }
+
+    if (!isBuffering) {
+        PlayPauseButton(player, showControls, modifier = modifier.align(Alignment.Center))
+    }
+    BottomControls(
+        player = player,
+        showControls = showControls,
+        isFullScreen = isFullScreen,
+        onClickFullscreen = onClickFullscreen,
+        modifier = modifier.align(Alignment.BottomCenter),
+    )
+}
+
+@Composable
+private fun BottomControls(
+    player: Player,
+    showControls: Boolean,
+    isFullScreen: Boolean,
+    onClickFullscreen: () -> Unit,
+    modifier: Modifier,
+) {
+    Row(
+        modifier = modifier
+            .height(60.dp)
+            .fillMaxWidth()
+            .background(
+                Brush.verticalGradient(
+                    colors = listOf(Color(0x30000000), Color(0xB0000000)),
+                ),
+            ),
+    ) {
+        TimeText(player)
+        Spacer(modifier = Modifier.weight(1f))
+        MuteButton(player, showControls)
+        FullscreenButton(isFullScreen, onClickFullscreen)
+    }
+}
+
+@Composable
+private fun RowScope.FullscreenButton(isFullScreen: Boolean, onClickFullscreen: () -> Unit) {
+    IconButton(
+        onClick = onClickFullscreen,
+        modifier = Modifier
+            .align(Alignment.CenterVertically)
+            .size(64.dp),
+    ) {
+        Icon(
+            if (isFullScreen) Icons.Default.FullscreenExit else Icons.Default.Fullscreen,
+            contentDescription = stringResource(R.string.fullscreen),
+            tint = Color.White,
+        )
+    }
+}
+
+@OptIn(UnstableApi::class)
+@Composable
+private fun RowScope.MuteButton(player: Player, showControls: Boolean) {
+    val muteState = rememberMuteUnmuteButtonState(player)
+    if (muteState.isEnabled) {
+        IconButton(
+            modifier = Modifier
+                .align(Alignment.CenterVertically)
+                .size(64.dp),
+            onClick = muteState::onClick,
+            enabled = showControls,
+        ) {
+            val icon =
+                if (muteState.showMute) Icons.AutoMirrored.Filled.VolumeMute else Icons.AutoMirrored.Filled.VolumeUp
```

Comment:
Might be personal preference, but `VolumeOff` seems like a better fit than `VolumeMute`, for example the system volume level also uses a slashed icon when at 0.

Reply:
You're right it looks better